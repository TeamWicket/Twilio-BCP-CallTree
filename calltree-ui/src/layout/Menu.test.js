import Menu from './Menu';
import React from 'react';
import { shallow } from 'enzyme';
import { Admin } from 'react-admin';

describe("Menu", () => {
    it("should match snapshot of ConfigurationMenu", () => {
        expect(shallow(<Admin><Menu /></Admin>)).toMatchSnapshot();
    })
})
