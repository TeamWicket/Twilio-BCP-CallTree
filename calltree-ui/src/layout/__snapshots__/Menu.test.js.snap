// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should match snapshot of ConfigurationMenu 1`] = `ShallowWrapper {}`;
